instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/messaging/MessagingAttributesGetter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter.messaging;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+// TODO: javadocs
```

Comment:
Can we at least add the class doc like we've done for the others?

Reply:
That was meant to be just a demonstration of the `AttributesGetter` idea - I'll revert it for now but we can always come back to this discussion later.